src/utils/getFilteredPeople.tsx

```diff
@@ -0,0 +1,64 @@
+import { Person } from '../types';
+
+const ZERO_YEARS = 1500 | 1600 | 1700 | 1800 | 1900 | 2000;
```

Comment:
The `ZERO_YEARS` constant is incorrectly defined using bitwise OR. This will not work as intended for checking centuries. Consider using an array or a different approach to handle century calculations.